src/rpcblockchain.cpp

```diff
@@ -77,6 +77,25 @@ double GetNetworkDifficulty(const CBlockIndex* blockindex)
     return GetDifficultyINTERNAL(blockindex, true);
 }
 
+static UniValue ValuePoolDesc(
+    const std::string &name,
+    boost::optional<CAmount> chainValue,
```

Comment:
should there be `const` infront of `chainValue,valueDelta` as they are just being read here?   (I think I managed to use `const` with boost::optional` )

Reply:
Oh, probably - I just copied this from another existing function, where this parameter was an `int`, so const didn't make sense.

Reply:
Great, if you could change this and answer my first comment I'll ack.